application/run-cron.sh

```diff
@@ -17,23 +17,8 @@ chown -R www-data:www-data \
   /data/frontend/web/assets/
 
 # Run database migrations
-output=$(/data/yii migrate --interactive=0 2>&1)
-
-# If they failed, exit.
-rc=$?;
-if [[ $rc != 0 ]]; then
-    logger --priority user.err --stderr "Migrations failed with status ${rc} and output: ${output}"
-    exit $rc;
-fi
-
-output=$(/data/yii migrate --interactive=0 --migrationPath=console/migrations-local 2>&1)
-# If they failed, exit.
-rc=$?;
-if [[ $rc != 0 ]]; then
-    logger --priority user.err "Migrations failed with status ${rc} and output: ${output}"
```

Comment:
yes, runny is our new wrapper, I didn't mean to include it in this PR but in a different one, but oh well, it can come in at the same time. See https://github.com/silinternational/runny